python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -262,6 +356,9 @@ def on_task_finished(self, task_index: int, exception: Optional[Exception]):
         task_info = self._running_tasks[task_index]
         self.num_outputs_of_finished_tasks += task_info.num_outputs
         self.bytes_outputs_of_finished_tasks += task_info.bytes_outputs
+        task_runtime = time.time() - task_info.start_time
```

Comment:
Is timing `on_task_submitted` -> `on_task_finished` good enough for task runtime? Or were you looking for only the time of the udf?